From 3648705ad86c292746efb7fcf2eeb7367e74b5bd Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Mon, 9 Dec 2024 17:24:26 +0800
Subject: [PATCH 203/286] AOSCOS: gpio: use CPU_LOONGSON64 for GPIO_NCT6102

Per commit 30ad29bb4888 ("MIPS: Loongson: Naming style cleanup and
rework"), rename the CPU_LOONGSON3 dependency as CPU_LOONGSON64.

Signed-off-by: Mingcong Bai <jeffbai@aosc.io>

Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
---
 drivers/gpio/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpio/Kconfig b/drivers/gpio/Kconfig
index e04b7a7870fd..408a91bb06af 100644
--- a/drivers/gpio/Kconfig
+++ b/drivers/gpio/Kconfig
@@ -524,7 +524,7 @@ config GPIO_NPCM_SGPIO
 config GPIO_NCT6102
 	bool "NCT6102 GPIO driver"
 	def_bool y
-	depends on CPU_LOONGSON3
+	depends on CPU_LOONGSON64
 	help
 	  Say yes here to support the NCT6102 GPIO device
 
-- 
2.48.1

